nilearn/image/image.py

```diff
@@ -665,3 +665,68 @@ def threshold_img(img, threshold, mask_img=None):
     threshold_img = new_img_like(img, img_data, affine)
 
     return threshold_img
+
+
+def math_img(formula, **imgs):
+    """Interpret a string formula using niimg in named parameters.
+
+    Parameters
+    ----------
+    formula: string
+        The mathematical formula to apply to image internal data.
+    imgs:
+        Keyword arguments corresponding to the variables in the formula.
+
```

Comment:
You may have missed adding documentation about "Returns" saying what this function returns.